rust_snuba/src/processors/spans.rs

```diff
@@ -32,33 +32,33 @@ pub fn process_message(
 }
 
 #[derive(Debug, Default, Deserialize, JsonSchema)]
-struct FromSpanMessage {
-    description: Option<String>,
-    duration_ms: u32,
-    #[serde(alias = "end_timestamp_micro")]
-    end_timestamp_precise: Option<f64>,
-    event_id: Option<Uuid>,
-    exclusive_time_ms: f64,
-    is_segment: bool,
-    measurements: Option<BTreeMap<String, FromMeasurementValue>>,
-    parent_span_id: Option<String>,
-    profile_id: Option<Uuid>,
-    project_id: u64,
-    received: f64,
-    retention_days: Option<u16>,
-    segment_id: Option<String>,
-    sentry_tags: Option<BTreeMap<String, String>>,
-    span_id: String,
+pub(crate) struct FromSpanMessage {
```

Comment:
Why do we need all this?

Reply:
spans and eap_spans both listen to the same kafka topic, so FromSpanMessage gets made crate public for use in both places